R/LearnerClustMiniBatchKMeans.R

```diff
@@ -69,7 +69,15 @@ LearnerClustMiniBatchKMeans = R6Class("LearnerClustMiniBatchKMeans",
       }
 
       pv = self$param_set$get_values(tags = "train")
-      invoke(ClusterR::MiniBatchKmeans, data = task$data(), .args = pv)
+      m = invoke(ClusterR::MiniBatchKmeans, data = task$data(), .args = pv)
+
+      self$assignments = unclass(ClusterR::predict_MBatchKMeans(
+        data = task$data(),
```

Comment:
This looks like a heavy computation is required. Is there a way to turn it off if we don't need the assignments?

Reply:
I added `save_assignments` field in `LearnerClust` that lets you turn off saving. It's on by default.